.browsersync.js

```diff
@@ -18,14 +18,14 @@ module.exports = {
             "port": 8080
         }
     },
-    "files": [ "examples/cdn/*"],
+    "files": [ "examples/bundle/*"],
```

Comment:
I don't think so.  Actually, when using browser-sync with the website/examples, it actually uses the `hexo-browsersync` plugin instead of any of this config.  We may not actually need tihs config file anymore, now that I think about it.